files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -24,6 +25,10 @@
 rabbitmq_data_dir_symlink = File.join(rabbitmq_dir, "db")
 rabbitmq_log_dir = rabbitmq['log_directory']
 
+# path needed for rabbitmqctl to run
+rabbitmq_path = rabbitmq['env_path']
+rabbitmq_env = ENV.to_hash.merge("PATH" => rabbitmq_path)
```

Comment:
Actually given this, it may be enough to just `/opt/opscode/bin:/opt/opscode/embedded/bin` in the attribute.